uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/components/my-leave-request.html

```diff
@@ -0,0 +1,177 @@
+<div class="modal-header">
+  <button type="button" class="close" ng-click="modal.closeModal()"><span aria-hidden="true">&times;</span><span
+    class="sr-only">Close</span></button>
+  <h4 class="modal-title">Leave Request</h4>
+</div>
+<div class="modal-body clearfix">
+  <div class="row">
+    <div class="col-xs-12">
+      <div class="chr_custom-select chr_custom-select--full">
+        <select>
+          <option value="1" selected>Vacation (15 days available)</option>
+          <option value="2">Medical</option>
+        </select>
+      </div>
+    </div>
+  </div>
+
+  <div class="row">
+    <div class="col-xs-12">
+      <uib-tabset class="chr-tab__outer">
+
+        <uib-tab heading="Details">
+          <div class="row">
+            <div class="col-xs-12 form-horizontal">
+              <div class="form-group">
+                <label class="col-xs-4 control-label">No. of Days</label>
+                <div class="col-xs-8">
+                  <div class="btn-group chr-toggle btn-group-justified" data-toggle="buttons">
```

Comment:
Same thing that I've said for `.chr_tab__outer`, here it seems you're just customizing `.btn-group`, then can't you just do that? Is it possible to imagine that every button group will have this styling?